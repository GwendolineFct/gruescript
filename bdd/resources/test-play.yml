---
  - name: a simple task
    realfacts:
  - name: another one
    breaking:
      param2: Don't go breaking my heart

  - name: and a last one
    registration:
    register: somevar
