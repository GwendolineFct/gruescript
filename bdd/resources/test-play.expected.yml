---
# *** MIG ***  WARNING : There is a breaking change in the facts returned.
# Please check documentation for latest version of module `migration.test.realfacts`
# https://docs.ansible.com/ansible/latest/collections/migration/test/realfacts_module.html
  - name: a simple task
    migration.test.realfacts:
  - name: another one
# *** MIG ***  ERROR : missing parameter `param1` is required in latest version
    migration.test.breaking:
# *** MIG ***  ERROR : unknown module parameter `param2` in latest version
      param2: Don't go breaking my heart

  - name: and a last one
    migration.test.registration:
# *** MIG ***  WARNING : There is a breaking change in the returned values
    register: somevar
